client/my-sites/customer-home/main.jsx

```diff
@@ -0,0 +1,212 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import CardHeading from 'components/card-heading';
+import EmptyContent from 'components/empty-content';
+import Main from 'components/main';
+import VerticalNav from 'components/vertical-nav';
+import VerticalNavItem from 'components/vertical-nav/item';
+import { preventWidows } from 'lib/formatting';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import { getSelectedSite, getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { getCustomizerUrl, canCurrentUserUseCustomerHome } from 'state/sites/selectors';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import DocumentHead from 'components/data/document-head';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const ActionBox = ( { action, iconSrc, label } ) => {
+	const buttonAction = 'function' === typeof action ? { onClick: action } : { href: action };
+	return (
+		<div className="customer-home__box-action">
+			<Button { ...buttonAction }>
+				<img src={ iconSrc } alt="" />
+				<span>{ label }</span>
+			</Button>
+		</div>
+	);
+};
+
+class Home extends Component {
+	static propTypes = {
+		site: PropTypes.object.isRequired,
+		siteId: PropTypes.number.isRequired,
+		siteSlug: PropTypes.string.isRequired,
+		customizeUrl: PropTypes.string.isRequired,
+		canUserUseCustomerHome: PropTypes.bool.isRequired,
+	};
+
+	render() {
+		const { translate, canUserUseCustomerHome } = this.props;
+
+		if ( ! canUserUseCustomerHome ) {
+			return (
+				<EmptyContent
+					title={ preventWidows( translate( 'You are not authorized to view this page.' ) ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'You are not authorized to view this page' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).